### PR TITLE
Top tables UI update

### PR DESCRIPTION
**Pull request by diamond-darrell**
Referring 5th item from the second table on the page https://kaltura.atlassian.net/wiki/spaces/PROD/pages/1153796355/Analytics+Phase+2+-+Enhancement

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -160,6 +160,8 @@
       "contributorsAmount": "How many users contributed more than once?",
       "returningContributors": "Returning Contributors",
       "noRanking": "Before April 2018 contributors ranking is not available",
+      "score": "Score",
+      "viewDetails": "View Details",
```
Comment (amirch1) [inline src/assets/i18n/en.json]: @diamond-darrell - should be "Contributor Details"

File: src/assets/i18n/en.json [hunk 2]
```diff
@@ -362,6 +364,8 @@
         "creationDate": "Creation Date",
         "duration": "Duration",
         "errorLoadingEntry": "Failed to load entry data",
+        "score": "Score",
+        "viewDetails": "View Details",
```
Comment (amirch1) [inline src/assets/i18n/en.json]: @diamond-darrell  - should be "Video Details"

Comment (diamond-darrell) [reply src/assets/i18n/en.json]: fixed

File: src/assets/i18n/en.json (hunk 1 above)
Comment (diamond-darrell) [reply src/assets/i18n/en.json]: fixed

**Pull request merged**